unity-client/Assets/Scripts/MainScripts/DCL/Components/BuilderInWorld/DCLName.cs

```diff
@@ -0,0 +1,38 @@
+using DCL;
+using DCL.Components;
+using DCL.Controllers;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// This component is a descriptive name of the Entity. In the BuilderInWorld you can give an entity a descriptive name through the entity list.
+/// Builder in World send a message to kernel to change the value of this component in order to assign a descriptive name
+/// </summary>
+public class DCLName : BaseDisposable
```

Comment:
Why is this component `Shared (Disposable)`? Can a same name be shared accross different entities?

Reply:
No, but the alternative involves forcing a name `MonoBehaviour`, and that is worse. We'll have to improve the design of the components in the future -_-